python/sdk/prelude_sdk/controllers/scm_controller.py

```diff
@@ -125,6 +125,102 @@ def update_evaluation(self, partner: Control):
             return res.json()
         raise Exception(res.text)
 
+    @verify_credentials
+    def list_object_exceptions(self):
+        """ List object exceptions """
+        res = self._session.get(
+            f'{self.account.hq}/scm/exceptions/objects',
+            headers=self.account.headers,
+            timeout=10
+        )
+        if res.status_code == 200:
+            return res.json()
+        raise Exception(res.text)
+
+    @verify_credentials
+    def create_object_exception(self, category: ControlCategory, filter, name = None, expires: str = None):
+        """ Create an object exception """
+        body = dict(
+            category=category.name,
+            filter=filter
+        )
+        if name:
+            body['name'] = name
+        if expires:
+            body['expires'] = expires
+        res = self._session.post(
+            f'{self.account.hq}/scm/exceptions/objects',
+            json=body,
+            headers=self.account.headers,
+            timeout=10
+        )
+        if res.status_code == 200:
+            return res.json()
+        raise Exception(res.text)
+
+    @verify_credentials
+    def update_object_exception(self, exception_id, expires = '', filter = None, name = None):
```

Comment:
why a string here but None in create?

Reply:
it's a weird/bad workaround to knowing if the CLI is passing through an 'expires' param that is actively None (ie. don't expire ever) vs not passing through expires at all (ie. leave the value alone)

could also make a BE change to read "9999-12-12" (datetime.max) as "never expire" and then None can can be read as "leave the value alone"... both are kinda awkward for different reasons

Reply:
hmmmmm i see, then should this also be changed to string in the put policy exception function? and the same check if expires == ''

Reply:
the policy exception is a full on put, right? expires should always be set and I'll just remove the default altogether

Reply:
yeah it's a full put

Reply:
are you also going to do the 'arg' thing?